src/assisted_test_infra/test_infra/utils/logs_utils.py

```diff
@@ -33,6 +39,49 @@ def _safe_open_tar(tar_path: Path | str, destination_path: Path | str) -> None:
         yield tar
 
 
+def _verify_duplicate_and_size_logs_file(dir_path: str) -> None:
+    """Verify logs files with different content and minimal size
+    Assuming all files under directory are file type
+    Checking:
+    Repeated names under directory
+    Repeated md5sum under directory
+    Included file name inside other files
+    Exceed limit size (1MB) under directory
+    Minimal file size (17 Bytes) when '-- No entries --'
+    :param dir_path:
+    :return: None
+    """
+    files = os.listdir(dir_path)
+    # assert if repeated file name exists
+    assert len(files) == len(set(files)), f"Repeated same file names {str(files)}"
```

Comment:
how to files in the same directory can have the same name? You have an example?

Reply:
Yes ... i know but see this bug : https://issues.redhat.com/browse/MGMT-13842

Reply:
I dont know how it happended but i added coverage for it , its  strange

Reply:
in the screenshot, the duplicated files are shown inside the tar? Once extracted, there will be only one file that "wins"? 
I think the duplicated names should be checked by inspecting the tar file, not after extracting it.

Reply:
ack